openlcs/packages/views.py

```diff
@@ -783,6 +786,15 @@ class PackageImportTransactionView(APIView, PackageImportTransactionMixin):
             "nvr": "rhel8.0",
```

Comment:
`package` no longer exists. I guess L785-L789 could be removed.

Reply:
Removed, thanks